Can't play a podcast

### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/AntennaPod/AntennaPod/issues) issues to see if someone else has already submitted the same bug report.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/AntennaPod/AntennaPod/issues?q=is%3Aissue+is%3Aclosed) issues to see if the problem is already solved and just waiting to be released.
- [X] I will describe the problem with as much detail as possible.
- [X] If the bug only to occurs with a certain podcast, I will include the URL of that podcast.

### App version

3.2.0

### Where did you get the app from

Google Play

### Android version

13 (lineageOS)

### Device model

_No response_

### First occurred

I just downloaded the app, tried multiple podcasts and none work.
Going to spotify it works fine.

### Steps to reproduce

1. Open a podcast
2. Press "stream"
3. Press the "play" button in the bottom right corner => doesn't do anything
4. Click the banner at the bottom of the screen
5. Click the play button there => closes the overview, and no more banner at the bottom
6. Download the podcast => works fine
7. Repeat above steps but with downloaded podcast => doesn't work either

### Expected behaviour

It should play

### Current behaviour

It doesn't play, and when pressing "play" from the screen where it shows advanced controls like going backwards/forwards/shownotes it will close the playing (seems like it's crashing?)

### Logs

"No crash report recorded"

I just checked logcat, and that gave the following error(s): 
```
31070 31070 W PlaybackController: Play/Pause button was pressed, but playbackservice was null!
31070 31070 D PlaybackService: Service created.
 1667  4156 I MediaSessionStack: addSession to bottom of stack | record: de.danoeh.antennapod/PlaybackService (userId=10)
31070 31070 D SessionManagerImpl: unimplemented Method: addSessionManagerListener
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@6665d8d already registered for pid 31070, callerPackage is de.danoeh.antennapod
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@224fb42 already registered for pid 31070, callerPackage is de.danoeh.antennapod
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@7f0d890 already registered for pid 31070, callerPackage is de.danoeh.antennapod
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@2e84589 already registered for pid 31070, callerPackage is de.danoeh.antennapod
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@1a705af already registered for pid 31070, callerPackage is de.danoeh.antennapod
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@c32edbc already registered for pid 31070, callerPackage is de.danoeh.antennapod
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@bab09a already registered for pid 31070, callerPackage is de.danoeh.antennapod
 1667  4156 W ActivityManager: Receiver with filter android.content.IntentFilter@2684dcb already registered for pid 31070, callerPackage is de.danoeh.antennapod
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D ItemDescriptionFragment: load()
31070 31070 D PlaybackService: OnStartCommand called
31070 31070 D PlaybackSrvState: startForeground
31070 31118 D DBReader: getFeedItem() called with: itemId = [1]
31070 31118 D DBReader: Loading feeditem with id 1
31070 31118 D DBReader: getFavoriteIDList() called
31070 31118 D DBReader: getQueueIDList() called
31070 31070 D PlaybackService: Received onBind event
31070 31118 D DBReader: Extracting Feedlist
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Trying to connect to service
31070 31070 D PlaybackController: Result for service binding: true
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D ItemDescriptionFragment: load()
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D PlaybackController: Querying service info
31070 31070 D PlaybackController: status: STOPPED
31070 31070 D ExternalPlayerFragment: Loading media info
31070 31070 D PlaybackController: Connection to Service established
31070 31070 D LclPlaybackSvcMPlayer: playMediaObject(...)
31070 31070 I ExoPlayerImpl: Init 77fda15 [ExoPlayerLib/2.14.2] [REDACTED]
  882  8595 I EffectsFactory: EffectCreate() create_effect
31070 31070 D PlaybackSvcMediaPlayer: LocalPSMP: Setting player status to INITIALIZING
 1667  1694 I MediaSessionStack: onPlaybackStateChanged - Pushing session to top | record: de.danoeh.antennapod/PlaybackService (userId=10)
31070 31070 W System.err: java.lang.IllegalArgumentException: User 10 is not the current user.
31070 31070 W System.err: 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3015)
31070 31070 W System.err: 	at android.os.Parcel.createException(Parcel.java:2995)
31070 31070 W System.err: 	at android.os.Parcel.readException(Parcel.java:2978)
31070 31070 W System.err: 	at android.os.Parcel.readException(Parcel.java:2920)
31070 31070 W System.err: 	at com.android.internal.statusbar.IStatusBarService$Stub$Proxy.requestTileServiceListeningState(IStatusBarService.java:2400)
31070 31070 W System.err: 	at android.app.StatusBarManager.requestTileServiceListeningState(StatusBarManager.java:906)
31070 31070 W System.err: 	at android.service.quicksettings.TileService.requestListeningState(TileService.java:518)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.QuickSettingsTileService$$ExternalSyntheticApiModelOutline0.m(R8$$SyntheticClass:0)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService$3.statusChanged(PlaybackService.java:856)
31070 31070 W System.err: 	at de.danoeh.antennapod.playback.base.PlaybackServiceMediaPlayer.setPlayerStatus(PlaybackServiceMediaPlayer.java:319)
31070 31070 W System.err: 	at de.danoeh.antennapod.playback.base.PlaybackServiceMediaPlayer.setPlayerStatus(PlaybackServiceMediaPlayer.java:331)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.LocalPSMP.playMediaObject(LocalPSMP.java:160)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.LocalPSMP.playMediaObject(LocalPSMP.java:113)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.startPlaying(PlaybackService.java:741)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.lambda$onStartCommand$6(PlaybackService.java:526)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.$r8$lambda$mOCf9qqsskaQGmxsaI_m2ZmJO2w(PlaybackService.java:0)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService$$ExternalSyntheticLambda6.accept(R8$$SyntheticClass:0)
31070 31070 W System.err: 	at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
31070 31070 W System.err: 	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:201)
31070 31070 W System.err: 	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255)
31070 31070 W System.err: 	at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124)
31070 31070 W System.err: 	at android.os.Handler.handleCallback(Handler.java:942)
31070 31070 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
31070 31070 W System.err: 	at android.os.Looper.loopOnce(Looper.java:201)
31070 31070 W System.err: 	at android.os.Looper.loop(Looper.java:288)
31070 31070 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7924)
31070 31070 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
31070 31070 W System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
31070 31070 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
31070 31070 W System.err: Caused by: android.os.RemoteException: Remote stack trace:
31070 31070 W System.err: 	at com.android.server.statusbar.StatusBarManagerService.requestTileServiceListeningState(StatusBarManagerService.java:1869)
31070 31070 W System.err: 	at com.android.internal.statusbar.IStatusBarService$Stub.onTransact(IStatusBarService.java:1259)
31070 31070 W System.err: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
31070 31070 W System.err: 	at android.os.Binder.execTransact(Binder.java:1244)
31070 31070 D PlaybackService: Playable was not found. Stopping service.
31070 31070 W System.err: java.lang.IllegalArgumentException: User 10 is not the current user.
31070 31070 W System.err: 	at android.os.Parcel.createExceptionOrNull(Parcel.java:3015)
31070 31070 W System.err: 	at android.os.Parcel.createException(Parcel.java:2995)
31070 31070 W System.err: 	at android.os.Parcel.readException(Parcel.java:2978)
31070 31070 W System.err: 	at android.os.Parcel.readException(Parcel.java:2920)
31070 31070 W System.err: 	at com.android.internal.statusbar.IStatusBarService$Stub$Proxy.requestTileServiceListeningState(IStatusBarService.java:2400)
31070 31070 W System.err: 	at android.app.StatusBarManager.requestTileServiceListeningState(StatusBarManager.java:906)
31070 31070 W System.err: 	at android.service.quicksettings.TileService.requestListeningState(TileService.java:518)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.QuickSettingsTileService$$ExternalSyntheticApiModelOutline0.m(R8$$SyntheticClass:0)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService$3.statusChanged(PlaybackService.java:856)
31070 31070 W System.err: 	at de.danoeh.antennapod.playback.base.PlaybackServiceMediaPlayer.setPlayerStatus(PlaybackServiceMediaPlayer.java:319)
31070 31070 W System.err: 	at de.danoeh.antennapod.playback.base.PlaybackServiceMediaPlayer.setPlayerStatus(PlaybackServiceMediaPlayer.java:331)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.LocalPSMP.playMediaObject(LocalPSMP.java:160)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.LocalPSMP.playMediaObject(LocalPSMP.java:113)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.startPlaying(PlaybackService.java:741)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.lambda$onStartCommand$6(PlaybackService.java:526)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService.$r8$lambda$mOCf9qqsskaQGmxsaI_m2ZmJO2w(PlaybackService.java:0)
31070 31070 W System.err: 	at de.danoeh.antennapod.core.service.playback.PlaybackService$$ExternalSyntheticLambda6.accept(R8$$SyntheticClass:0)
31070 31070 W System.err: 	at io.reactivex.internal.observers.LambdaObserver.onNext(LambdaObserver.java:63)
31070 31070 W System.err: 	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(ObservableObserveOn.java:201)
31070 31070 W System.err: 	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(ObservableObserveOn.java:255)
31070 31070 W System.err: 	at io.reactivex.android.schedulers.HandlerScheduler$ScheduledRunnable.run(HandlerScheduler.java:124)
31070 31070 W System.err: 	at android.os.Handler.handleCallback(Handler.java:942)
31070 31070 W System.err: 	at android.os.Handler.dispatchMessage(Handler.java:99)
31070 31070 W System.err: 	at android.os.Looper.loopOnce(Looper.java:201)
31070 31070 W System.err: 	at android.os.Looper.loop(Looper.java:288)
31070 31070 W System.err: 	at android.app.ActivityThread.main(ActivityThread.java:7924)
31070 31070 W System.err: 	at java.lang.reflect.Method.invoke(Native Method)
31070 31070 W System.err: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
31070 31070 W System.err: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
31070 31070 W System.err: Caused by: android.os.RemoteException: Remote stack trace:
31070 31070 W System.err: 	at com.android.server.statusbar.StatusBarManagerService.requestTileServiceListeningState(StatusBarManagerService.java:1869)
31070 31070 W System.err: 	at com.android.internal.statusbar.IStatusBarService$Stub.onTransact(IStatusBarService.java:1259)
31070 31070 W System.err: 	at android.os.Binder.execTransactInternal(Binder.java:1280)
31070 31070 W System.err: 	at android.os.Binder.execTransact(Binder.java:1244)
31070 31070 D PlaybackSrvState: stopService
31070 31070 D PlaybackSrvState: stopForeground
31070 31070 D PlaybackController: Disconnected from Service
31070 31070 D PlaybackController: Disconnected from Service
31070 31070 D PlaybackController: Disconnected from Service
31070 31070 D PlaybackService: Received onUnbind event
31070 31070 D PlaybackService: Service is about to be destroyed
31070 31070 D PlaybackSrvState: stopForeground
31070 31070 D SessionManagerImpl: unimplemented Method: removeSessionManagerListener
 1667  1694 I MediaSessionStack: removeSession | record: de.danoeh.antennapod/PlaybackService (userId=10)
31070 31070 I ExoPlayerImpl: Release 77fda15 [ExoPlayerLib/2.14.2] [REDACTED] [goog.exo.core]
 1667  1694 I MediaFocusControl: abandonAudioFocus() from uid/pid 1010233/31070 clientId=android.media.AudioManager@53a7464de.danoeh.antennapod.core.service.playback.LocalPSMP$1@2b236cd
31070 31070 D PlaybackController: Disconnected from Service
31070 31070 D PlaybackController: Disconnected from Service
```

And this in particular: `User 10 is not the current user`. Since I'm running AntennaPod in a android work profile (see the [shelter](https://f-droid.org/packages/net.typeblog.shelter/) application), the user ID is 10. I'm not sure what exactly is going on, but hopefully this can send you in the right direction. Let me know if there's something else I can do.

--- a/common.gradle
+++ b/common.gradle
@@ -44,7 +44,7 @@ android {
     }
 
     lint {
-        disable "GradleDependency"
+        disable "GradleDependency", "OutdatedLibrary"
         checkDependencies true
         warningsAsErrors true
         abortOnError true

--- a/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java
+++ b/core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java
@@ -867,8 +867,12 @@ public class PlaybackService extends MediaBrowserServiceCompat {
             }
 
             if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
-                TileService.requestListeningState(getApplicationContext(),
-                        new ComponentName(getApplicationContext(), QuickSettingsTileService.class));
+                try {
+                    TileService.requestListeningState(getApplicationContext(),
+                            new ComponentName(getApplicationContext(), QuickSettingsTileService.class));
+                } catch (IllegalArgumentException e) {
+                    Log.d(TAG, "Skipping quick settings tile setup");
+                }
             }
 
             IntentUtils.sendLocalBroadcast(getApplicationContext(), ACTION_PLAYER_STATUS_CHANGED);

